bin/cmd.js

```diff
@@ -427,8 +430,10 @@ function toStamp (s) {
 }
 
 function getKey (x) {
+    if (x === 'NaN') return strftime(KEY_FORMAT, new Date(NaN));
```

Comment:
Hm, this will always return `'time!NaN-NaN-NaN NaN:NaN:NaN'`...? Is this intended?

Reply:
Yes. Working around the problem where that key ends up persisted in the db by enabling you to directly address that [broken] record. See my comments on Issue #32.

Reply:
Wouldn't it be better to avoid `NaN` records in the first place? :)

Reply:
Certainly. However, I already had a corrupted record with the NaN stamp. Until a complete fix is worked out, this at least lets the problem be corrected.